docs/surveys/2020-jupyterlab-survey.md

```diff
@@ -0,0 +1,363 @@
+
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+### Collaboration
+
+16. With how many other people are you collaborating?
+    - 0
+    - 10
+    - 20
+    - 30
+    - 40
+    - 50+
+
+17. What is your reason for sharing a notebook with someone else? (up to 3)
+    - ❗I am not working with other people.
+    - Share knowledge.
+    - Feedback about my writing.
+    - Feedback about my code.
+    - Formal code review.
+    - Integrate my code/ data with their downstream or upstream processes.
+    - Edit/ contribute some of their own code.
+    - Edit/ contribute some of their own writing.
+    - Teach/ tutor them.
+    - Peer programming.
+    - Deploy my code/ model/ pipeline/ dashboard.
+    - Other (please specify)
+
+18. What is the nature of your collaboration?
+    - Describe the collaboration:
+        - How long have you been working together?
+            - I am not collaborating.
+            - 2+ years.
+            - 1-2 years.
+            - 6-12 months.
+            - Less then 6 months.
+        - How frequently do you work together?
+            - I am not collaborating.
+            - 2+ times per week.
+            - Weekly.
+            - A few times a month.
+            - Monthly.
+            - Less then monthly.
+        - How do you divide the work?
+            - I am not collaborating.
+            - We work on different projects.
+            - We work on the same project, but different parts.
+            - We work on the same part of the same project together.
+    - Comments about collaboration:
+
+19. Do you have challenges with collaboration in Jupyter?
+    - Don't know what dependencies (versions of language, packages, extensions) a notebook uses.
+    - Don't know/ have the data a notebook is supposed to use.
+    - Poor support for our version control (git) system.
+    - Publishing my notebook to a shared location.
+    - Not being able to comment on notebooks.
+    - More robust non-git version history of a notebook.
+
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
+20. Do you have challenges with the notebook UI?
+    - Poor autocompletion [LSP, show methods/ attributes].
+    - No modes for editing other Jupyter documents (MyST, Jupyter Book).
+    - Can't see hidden (.) files in file browser.
+    - Can't collapse sections of a notebook hierarchically.
+    - No progress bar for running long notebooks.
+    - Don't know which cell failed in long notebook.
+    - No Extension marketplace (5 star ratings, categories).
+    - No global search.
+    
+    For each of the items above, specify:
+    - Not a problem for me.	
+    - Trivial.	
+    - Minor.	
+    - Major.	
+    - Critical.	
+    - N/A - skip, don't know.
+
```

Comment:
I know we are on voting stage but I was wondering about the mobile story. Would it make sense to add a question on what are the expectations of Jupyter users regarding mobile, or something like that in terms of the different areas?

Reply:
Mobile use (desired or attempted) would be very interesting to find out. We've long said that we were not focusing on mobile with jlab, however some people have made things work to some degree. If there is a huge community interest in having jlab work on mobile, that would be very interesting.

Reply:
Hmm. One approach would be to add mobile devices to the "How do you access Jupyter" question with a callout for 'comments welcome about the experience.'

Reply:
![image](https://user-images.githubusercontent.com/16389820/98454852-3588ae80-2137-11eb-82e6-0ece207e8b1c.png)
☑️ changes integrated